stdlib/email/__init__.pyi

```diff
@@ -1,17 +1,21 @@
+# pyright: reportInvalidTypeVarUse=false
```

Comment:
See https://github.com/microsoft/pyright/discussions/8084 for some context on why we need this.